core/src/main/resources/jenkins/security/ApiTokenProperty/config.properties

```diff
@@ -18,3 +18,4 @@ RevokedToken=The legacy token has been revoked
 NoCreationDate=There is no creation date for that token
 NoCreationDateValue=Unknown
 ConfirmRevokeSingleTitle=Revoke token
+tokenExpiresOn=Expires on {0}
```

Comment:
What about when the token is expired? How does that look in the UI?

Reply:
yep, some that is missing so far